Hangman/Hangman/main.swift

```diff
@@ -8,5 +8,126 @@
+
+
+var playerIsPlaying = true
+var playerChoice = ""
+var playerIsNotPlaying = false
+
+print("Welcome to Hangman!!")
+sleep(1)
+
+print("Would You Like To Get Started ?! Type \"Yes\" or \"No\"")
+
+playerChoice = readLine()?.lowercased() ?? ""
+
+if playerChoice == "yes" {
+    playerIsPlaying = true
+    print("Awesome!!")
+} else if playerChoice == "no" {
+   
+    print("See You next Time!")
+    playerIsNotPlaying = false
+}
+
+while playerIsPlaying {
+    
+    let mysteryWord = allTheWords.randomElement() ?? ""
+
+    var hiddenWord: [Character] = Array(repeating: "_", count: mysteryWord.count)
+
+
+    print("You have 6 tries to guess the mystery Word")
+
+    sleep(2)
+
+    print(String(hiddenWord))
+    //print(mysteryWord) // code this out later
+
+
+
+
+    var userInput = ""
+
+    var guessMaximum = 6
+
+    var indices: Set<Int> = []
+    var foundWord = ""
+    var playerIsStillPlaying = true
+
+    var correctGuess = false
+
+    print("Are You Ready?! Guess a Letter!")
+
+
+repeat  {
+    
+userInput = readLine()?.lowercased() ?? ""
+    
+//checks if userr input is equal to character in random element
+    
+ for (index,char) in mysteryWord.enumerated() {
+
+            if userInput == String(char) {
+        
+                indices.insert(index)
+                
+                correctGuess = true
+                //print("Great!")
+                
+    }
+}
+    // tells player if guess is wrong and decreases guess.
+    
+    if correctGuess == false {
+        guessMaximum-=1
+        print ("Sorry! Wrong letter. You have \(guessMaximum) guesses left! ")
+        
+    }
+    // replaces "_" with userinput if its correct
+            for (index,char) in hiddenWord.enumerated() {
+    
+                if indices.contains(index) && char == Character("_") {
+                    
+                    
+                    hiddenWord[index] = Character(userInput)
+                    
+                    
+                    foundWord = String(hiddenWord)
+                    
+                    
+                }
+
+            
+            }
+   print(foundWord)
+    
+    correctGuess = false
+    //print("\(indices.count) is the same as \(mysteryWord.count)")
+}
+    
+    while guessMaximum > 0 && (indices.count < mysteryWord.count)
+
+    if indices.count == mysteryWord.count  {
+        print("AWESOME!!! 🥳🥳🥳🥳🥳🎉🥳🥳🥳🥳🥳 YOU WIN!!!")
```

Comment:
You have your winning statement but you also needed a loosing statement that would display the random word as well as give the user the opportunity to play again.